recipes/marray/bld.bat

```diff
@@ -5,6 +5,7 @@ cmake ^
     -G "%CMAKE_GENERATOR%" ^
     -DCMAKE_PREFIX_PATH=%LIBRARY_PREFIX% ^
     -DCMAKE_INSTALL_PREFIX=%LIBRARY_PREFIX% ^
+    -DHDF5_C_LIBRARIES=%LIBRARY_PREFIX%\Library\lib\hdf5.lib ^
```

Comment:
Ah right, thanks. For some reason CMake's HDF5 module is pickling up the ``.dll`` instead of the ``.lib``, so I was trying to force it to use the ``.lib``. Do you know anything about that?